free-1.md

```diff
@@ -11,41 +13,98 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+There is a vast array of television shows and movies where no subtitles are provided or only exist in the show’s language. Fansubbing has historically been the avenue where these types of videos gain subtitles. Viewers participating in these fandom spaces often want to learn a foreign language. These learners can be fan translators who want to improve their skills in the language.
+
+Dual-language (or multi-language) subtitles can help viewers optimize their learning by visually pairing the dialogue in the show’s language with the viewer’s native language. Fan translators can also exercise their skills in multiple languages. However, free subtitling tools have a relatively steep learning curve, are often time-consuming to use, and generally require a team of fansubbers that specialize in different aspects of subtitling - the “timers”, who create the template with proper timestamps; the “translators”, who write the line-by-line translation within the template; and the “editor/encoder”, who puts the subtitle together with the video. 
+
+Implementing dual-language subtitles puts twice the amount of time for timers and translators for a single video. Since this labor is unpaid and voluntary, dual-language subtitling is not commonly seen in fansubbed videos.
+
+A tool that alleviates tensions between free subtitling software and fansubbers and between creating different sets of subtitles can promote dual-language subtitles in fan-translated videos to help language-learning fans and create extra learning opportunities for the fan translators.
+
+
```

Comment:
I am already a fan of this topic for a DSL, it seems very niche and like it would solve a real world problem